utils/utils.go

```diff
@@ -357,3 +358,24 @@ func DecodeStructToMap(input interface{}) (map[string]interface{}, error) {
 func IsCamelCaseEqualToSnakeCase(a, b string) bool {
 	return strings.EqualFold(strcase.ToSnake(a), b)
 }
+
+// ReadPublicKey Reads public key from file or directly provided and returns key string if valid
+func ReadPublicKey(pathOrKey string) (string, error) {
+	var bytes []byte
+	var err error
+	if CheckFileExists(pathOrKey) {
+		bytes, err = os.ReadFile(pathOrKey)
+		if err != nil {
+
+			return "", err
+		}
```

Comment:
```suggestion
		if bytes, err = os.ReadFile(pathOrKey); err != nil {
			return "", err
		}
```

Reply:
don't forget to delete the old variant too, now there's both of them
we can actually shorten this even more if we overwrite the byte slice if a file has been supplied
```suggestion
	var err error
	bytes := []byte(pathOrKey)

	if CheckFileExists(pathOrKey) {
		log.Printf("[DEBUG] ssh key has been provided in the following file: %s", pathOrKey)
		if bytes, err = os.ReadFile(pathOrKey); err != nil {
			return "", err
		}
	}
	# rest unchanged
```

Reply:
@cbeti-ionos it will be solved in the commit after c9df22e0066ca69a7809de59cd124fe97fec9580, I left it as you suggested in the first place in order to keep the message in the logs.